modules/extensions/scripts/install_kubectx.template.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2022 Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+curl -sSfL https://github.com/ahmetb/kubectx/releases/download/v0.9.4/kubectx_v0.9.4_linux_x86_64.tar.gz -o kubectx_v0.9.4_linux_x86_64.tar.gz
+curl -sSfL https://github.com/ahmetb/kubectx/releases/download/v0.9.4/kubens_v0.9.4_linux_x86_64.tar.gz -o kubens_v0.9.4_linux_x86_64.tar.gz
+
+tar zxf  kubectx_v0.9.4_linux_x86_64.tar.gz
+tar zxf kubens_v0.9.4_linux_x86_64.tar.gz
+rm -f  kubectx_v0.9.4_linux_x86_64.tar.gz
+rm -f kubens_v0.9.4_linux_x86_64.tar.gz
+rm -f LICENSE
+
```

Comment:
I recommend parameterisaing the version so you don't have to edit all the commands every time it updates, just the version variable. Also, the repo supports multiple operating systems and architectures, so let's plan for the future and enable that too:

```bash
KUBECTX_VERSION="0.9.4"
KUBENS_VERSION="0.9.4"
ARCH=$(uname -m)
```
Then, you could combine all of this into two commands:

```bash
curl -sSfL https://github.com/ahmetb/kubectx/releases/download/v${KUBECTX_VERSION}/kubectx_v${KUBECTX_VERSION}_linux_${ARCH}.tar.gz | tar zxvf - -C /usr/local --xform='s|^|kubectx/|S'
curl -sSfL https://github.com/ahmetb/kubectx/releases/download/v${KUBENS_VERSION}/kubens_v${KUBENS_VERSION}_linux_${ARCH}.tar.gz | tar zxvf - -C /usr/local --xform='s|^|kubens/|S'
```

This will create `/usr/local/kubectx/{kubectx,LICENSE}` and `/usr/local/kubens/{kubens,LICENSE}` and retains the `LICENSE` for each tool, which is mandatory. You can link them to `/usr/local/bin` and alias as needed.

Reply:
I had this before but Terraform then expects me to pass the value. Ditto for ARCH. I've changed this somewhat. Please check again.